src/convertToObject.js

```diff
@@ -5,8 +5,25 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(str) {
+  const resultStyles = {};
```

Comment:
The variable name 'resultStyles' could be more descriptive. Consider a name that reflects the nature of the data it holds, such as 'styleObject' or 'styleMap' to improve code readability.